bin/commands/execute.js

```diff
@@ -11,8 +11,15 @@ import { store } from "../../src/localStorage.js";
 import { fetchPackagesAndRepositoriesData } from "../../src/analysis/fetch.js";
 import * as CONSTANTS from "../../src/constants.js";
 import * as reporting from "../../src/reporting/index.js";
+import { debug } from "../options/debug.js";
+
+export async function execute(options) {
+  const debugMode = options?.debug || false;
```

Comment:
```suggestion
export async function execute(options = {}) {
  const { 
    debug = Boolean(process.env.DEBUG)
  } = options;
```

I propose to use `process.env.DEBUG` by default as it is a standard var to enable/disable debug mode. (if so we should document it in the README)

Note: destructuring is the syntax we use everytime, almost a convention ^^ It scale better if we add more options in the future

Reply:
Optional chaining was not relevant as the object is always present as parameter, do I still have to give an empty object ?

In addition, I'm not sure about your proposition because if we run 
`DEBUG=http nreport exec`

Everything related to the debug option will be enabled while it has not been specified by the user.
It works only when DEBUG var is false.

Reply:
I think we want an empty object by default yes.
For the DEBUG var, I thought to a boolean value, we can do `process.env === "true"` to be more accurate, `debug` npm package should not be supported

Reply:
I understand, you want to fallback to a debug var that may exists in the .env file, right ?

But is it not the same behavior as the DEBUG npm package ? Both will read process.env file.

Reply:
I personnaly have no opinion about that, let defined it to `false` by default on sade and use destruct then it's ok for me.